adding_species_manuscript.md

```diff
@@ -107,133 +126,120 @@ since it is these patterns of polymorphism themselves that we want to compare to
 The scale over which linked selection has an effect in practice
 can differ greatly depending on the species and the context,
 and the actual extent is unknown.
-Even in large-population-size species,
-recent selective sweeps (e.g., insecticide resistance XXX)
+Even in species with large population sizes,
+recent selective sweeps (e.g., insecticide resistance TODO:CITE)
 can cause large haplotypes to reach high frequency.
-Another practical consideration is demographic realism:
-the genetic load in a simulation of a small segment of genome with deleterious mutations
+
+Third, linkage has the potential to affect demographic realism:
+the genetic load in a simulation of a small segment of chromosome with deleterious mutations
 will necessarily be less than that in whole chromosome;
 this makes it easy to simulate with unrealistically high levels of load
 without realizing it if small segments are used.
+
+However it is much faster to simulate many small chunks of genome, in part because it effectively parallelizes the simulation.
 It's probably safe to say that independently simulating small segments
 will not introduce serious problems in some situations, but could cause serious biases in others.
 The degree to which it is important to include linked selection in whole-chromosome simulations
 is, we think, still a major open question in the field.
 
-Finally, what about whole *genome* simulations?
+Finally, what about *whole genome* simulations?
 Chromosomes segregate independently,
 so between-chromosome correlations are possible but can only be induced by fairly extreme situations.
-For this reason, we tend to simulate chromosomes independently,
-and do not have an easy mechanism to simulate multiple chromosomes simultaneously
-with stdpopsim.
+For this reason, we tend to simulate chromosomes independently, and few (TODO:any? if so, find them)
+simulators have mechanisms to simulate multiple chromosomes simultaneously.
 
-
-## Making a popgen simulation
+## Making a population genomic simulation
 
 Purposes of generating population genetic simulations include
 testing and benchmarking inference methods,
 creating data for Approximate Bayesian Computation and for training machine learning methods,
-and (TODO: should complement the introduction).
+and providing comparative data for hypothesis testing, such as null models in selection analyses.
 These applications are more reliable and robust
 the more aspects of realism can be incorporated into the simulations,
 including linkage, recombination rate variation, demographic changes, and natural selection.
 So when choosing to generate realistic population genetic simulations for a species of interest, certain elements should be known.
-Regardless of the simulation program used,
-to run a population genetics simulation
-with the current standard of realism,
-we would ideally have for the species:
+To run a population genetics simulation with the current standard of realism, 
```

Comment:
I'm going to leave this one as-is for now. I'm not sure that "best approximates" is the best (hah) alternative since what is best is subject to change quickly. "standard of realism" is vague though.